### DISCUSSION TITLE
请问后续可否添加SMB或者CIFS类协议支持呢

### DISCUSSION THREAD

**Discussion by isjason** [AlistGo/alist#2166] [General]
后续可否添加SMB或者CIFS类协议支持呢？有些音影软件只支持smb协议，不支持挂载webdav，如果后续能支持SMB之类的其它协议就好了

**Comment by goldsoft8888** [comment]
同样需求，并且还想问一下。收费版可以映射本地盘，那能共享出去吗？能安装在路由器里吗？
alist 3.28版本非常棒，不占用CPU、内存、磁盘，效果很好。
用rclone 挂载ALIST, 缓存占用太大，会把路由器拖慢，
